tests/Http/Api/ActivityApiTest.php

```diff
@@ -208,20 +208,18 @@ public function testActivityIndexWithUpdatedAtQuery()
 
         // Create two signups and two posts. Associate the signups and posts respectively.
         $firstSignup = factory(Signup::class)->create();
-        $firstPost = factory(Post::class)->create();
-        $firstPost->signup()->associate($firstSignup);
+        $firstSignup->posts()->save(factory(Post::class)->make());
```

Comment:
ooo this is so nice!

Reply:
since the Post model factory now returns a signup id, will this conflict here when we associate these? I think the Post model factory also creates a signup (returns a newly created `signup_id`) so I'm wondering if this will clash?

Reply:
I believe it should only evaluate the closure in the Post factory if a value isn't already provided for `signup_id`, which it should be here. Lemme double-check.